__tests__/schema.sql

```diff
@@ -235,6 +278,67 @@ begin
 end;
 $$;
 ALTER FUNCTION graphile_worker.get_job(worker_id text, task_identifiers text[], job_expiry interval, forbidden_flags text[]) OWNER TO graphile_worker_role;
+CREATE FUNCTION graphile_worker.get_jobs(worker_id text, task_identifiers text[] DEFAULT NULL::text[], job_expiry interval DEFAULT '04:00:00'::interval, forbidden_flags text[] DEFAULT NULL::text[]) RETURNS SETOF graphile_worker.jobs
+    LANGUAGE plpgsql
+    AS $$
+declare
+  v_now timestamptz = now();
+begin
+  if worker_id is null or length(worker_id) < 10 then
+    raise exception 'invalid worker id';
+  end if;
+  return query with jobs_q as (
+    select
+      id, queue_name
+      from "graphile_worker".jobs jobs
+      where (jobs.locked_at is null or jobs.locked_at < (v_now - job_expiry))
+      and (
+        jobs.queue_name is null
+      or
+        exists (
+          select 1
+          from "graphile_worker".job_queues
+          where job_queues.queue_name = jobs.queue_name
+          and (job_queues.locked_at is null or job_queues.locked_at < (v_now - job_expiry))
+          for update
+          skip locked
+        )
+      )
+      and run_at <= v_now
+      and attempts < max_attempts
+      and (task_identifiers is null or task_identifier = any(task_identifiers))
+      and (forbidden_flags is null or (flags ?| forbidden_flags) is not true)
+      order by priority asc, run_at asc, id asc
+      -- TODO make this a configurable value
+      limit 1
```

Comment:
Make the job_count the first argument to `get_jobs`.

Also maintaining both get_job and get_jobs is likely to be a pain; lets switch worker to using `get_jobs` (assuming no performance cost) and replace `get_job(...)` with a call to `get_jobs(1, ...)` for legacy support.